recipes/kuba-zip/all/CMakeLists.txt

```diff
@@ -0,0 +1,9 @@
+cmake_minimum_required(VERSION 3.4)
+project(cmake_wrapper)
+
+include("${CMAKE_BINARY_DIR}/conanbuildinfo.cmake")
+conan_basic_setup()
+
+set(CMAKE_WINDOWS_EXPORT_ALL_SYMBOLS ON)
```

Comment:
OK.

Surprinsingly, upstream CMakeLists calls generate_export_headers, but don't use generated file 😩

Seems like author tried something, then mixed different solutions: https://github.com/kuba--/zip/pull/122

Reply:
yeah, maybe author is not so familiar with CMake

Reply:
I've submitted https://github.com/kuba--/zip/pull/178